lib/ecto/adapters/sqlite3.ex

```diff
@@ -51,6 +51,12 @@ defmodule Ecto.Adapters.SQLite3 do
     * `:uuid_type` - Defaults to `:string`. Determines the type of `:uuid` columns.
       Possible values and column types are the same as for
       [binary IDs](#module-binary-id-types).
+    * `:map_type` - Defaults to `:string`. Determines the type of `:map` columns.
+      Set to `:binary` to use the [JSONB](https://sqlite.org/draft/jsonb.html)
```

Comment:
Let's point at the official docs and not the draft.

```suggestion
      Set to `:binary` to use the [JSONB](https://sqlite.org/jsonb.html)
```

Reply:
Oops I missed that, fixed!